packages/roosterjs-editor-core/lib/coreApi/insertNode.ts

```diff
@@ -174,7 +176,7 @@ export const insertNode: InsertNode = (core: EditorCore, node: Node, option: Ins
                         pos = new Position(blockElement.getEndNode(), PositionType.After);
                     } else {
                         adjustSteps.forEach(handler => {
-                            pos = handler(contentDiv, node, pos);
+                            pos = handler(contentDiv, node, pos, core);
```

Comment:
Do not pass editorCore out of roosterjs-editor-core package. Please check what do these handlers need from editor core, and pass them separately.